Base.rte/AI/HumanAI.lua

```diff
@@ -1,10 +1,10 @@
 dofile("Base.rte/Constants.lua")
 require("AI/NativeHumanAI")
-require("AI/HumanFunctions");
+require("AI/HumanFunctions")
```

Comment:
tiny stupid nitpick: shouldn't these all have semi-colons after em? theyre used elsewhere in the code

or is the practice not to semicolon for requires and dofiles?

Reply:
I dunno but since there was none there before, I removed it instead of adding the semicolons everywhere. They're widely unused in the AI files for whatever reason

Reply:
Lua doesn't care but I do so technically they should. That said, adding it everywhere in the ai files when they're gonna be rewritten from the ground up is a waste of effort, so it's fine as-is.